src/components/CustomMessage.tsx

```diff
@@ -58,12 +58,12 @@ export default function CustomMessage(props: Props) {
   const { userId } = useWidgetLocalStorage();
 
   // admin message
-  if (message.messageType === 'admin') {
+  if (message.isAdminMessage()) {
     return <div>{<AdminMessage message={message} />}</div>;
   }
 
   if (isFormMessage(message)) {
-    const forms = message.extendedMessagePayload.forms;
+    const forms: Form[] = message.extendedMessagePayload!.forms as Form[];
```

Comment:
Any reason we force type casting here in this way?

Reply:
because forms is unknown type so `forms[0]` complains

Reply:
Is it different? `message.forms` vs `message.extendedMessagePayload.forms`

Reply:
maybe now, we can remove assertion (`as Form[]`)